config.json

```diff
@@ -234,6 +234,14 @@
         ],
         "status": "beta"
       },
+      {
+        "name": "The Farm",
+        "slug": "the-farm",
+        "uuid": "43e12a9e-ed8e-4552-a889-d86e284b21c0",
+        "concepts": [],
+        "prerequisites": [],
+        "status": "wip"
+      },
```

Comment:
Why did you add this entry? Was it to fix CI?

Reply:
I did not add this. I merged `main` into this branch. This entry is already on `main` [here](https://github.com/exercism/go/blob/main/config.json#L237-L244).